api/routers/cameras.py

```diff
@@ -143,6 +144,23 @@ async def get_camera(camera_id: str):
     return camera
 
 
+def get_first_unused_id(cameras_ids):
+    if not cameras_ids:
+        return 0
+
+    is_a_number = re.compile("^[0-9]+$")
+    ids_numbers = [int(value) for value in cameras_ids if is_a_number.match(value)]
+
+    if not ids_numbers:
+        return 0
+
+    ids_numbers.sort()
+    for i in range(0, ids_numbers[len(ids_numbers)-1]+1):
+        if ids_numbers[i] != i:
+            return i
+    return ids_numbers[len(ids_numbers)-1] + 1
```

Comment:
Can't you do `ids_numbers[-1] + 1` or is that just with numpy?

Reply:
I think you can

Reply:
Yes, you can